yarn-project/circuits.js/src/structs/kernel/__snapshots__/index.test.ts.snap

```diff
@@ -568,6 +570,8 @@ public_inputs: call_context:
 msg_sender: 0x1022
 storage_contract_address: 0x1023
 portal_contract_address: 0x1024
+function_selector: value: 1025
```

Comment:
I don't think it matters but:
In the diff above, `portal_contract_address` is 0x1014 (or 4116 in decimal). So `function_selector` was 4117.

Here, 0x1024 is 4132. So `function selector` should have been 4133 no? Same for the other diffs in this file

Again unsure if it matters or if it hints at a problem

Reply:
The function_selector is not a hex, because it is number underlying (due to its limited size). Maybe I should just make it a field here as well for consistency though 🤔